app/auth/login.tsx

```diff
@@ -30,6 +32,25 @@ export const links: LinksFunction = () => {
 export const meta: MetaFunction = () => {
   return { title: "Webstudio Login" };
 };
+const isPreviewDeployment = env?.VERCEL_ENV !== "preview";
```

Comment:
env is not optional, its always an object, right?

Reply:
also I don't get the logic, it says its not preview but the variable isPreview

Reply:
That was for testing, need to revert when it's working

Reply:
Fixed both